utils/EventTrigger.cpp

```diff
@@ -73,6 +73,9 @@ namespace Kernel
     const EventTrigger& EventTrigger::PFA_SeekingPartner             = EventTriggerFactory::CreateBuiltInEventTrigger( "PFA_SeekingPartner"             );
     const EventTrigger& EventTrigger::PFA_FoundPartner               = EventTriggerFactory::CreateBuiltInEventTrigger( "PFA_FoundPartner"               );
     const EventTrigger& EventTrigger::PFA_NoPartnerFound             = EventTriggerFactory::CreateBuiltInEventTrigger( "PFA_NoPartnerFound"             );
+    const EventTrigger & EventTrigger::MaritalCondomlessSexAct       = EventTriggerFactory::CreateBuiltInEventTrigger( "MaritalCondomlessSexAct"        );
+    const EventTrigger & EventTrigger::NonMaritalCondomlessSexAct    = EventTriggerFactory::CreateBuiltInEventTrigger( "NonMaritalCondomlessSexAct"     );
+
```

Comment:
nit: formatting

Reply:
@stitova-idm I don't understand your comment.  I think the formatting looks great.

Reply:
![image](https://github.com/user-attachments/assets/b7724326-8b37-4127-a89a-16bf7d2ab6ed)

Reply:
didn't see that.  not going to hold up for that